<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextItemDialog</class>
 <widget class="QWidget" name="TextItemDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>242</width>
    <height>143</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Text Item</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Font</string>
   </property>
   <widget class="QPushButton" name="clrButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="sizeSpinBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>8</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QFontComboBox" name="fontComboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>TextItemDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>TextItemDialog</receiver>
   <slot>clickedOkButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clrButton</sender>
   <signal>clicked()</signal>
   <receiver>TextItemDialog</receiver>
   <slot>clickedColorButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>clickedOkButton()</slot>
  <slot>clickedColorButton()</slot>
 </slots>
</ui>
